### ISSUE TITLE
Training custom datasets

### ISSUE THREAD

**Issue by johannes-do** [balancap/SSD-Tensorflow#94]
I have a custom dataset which I would like to train. Problem is, the loss just stays constant (constantly bad) for a long time, and after even longer the loss goes to infinity. The accuracy of the model is always around 0%. Any ideas why this could be?
Can I train end to end? Or do I need to train the classifier (the base network) first and when training the ssd model initialize the model with the previous trained weights?

**Comment by dlazares** [comment]
@j-o-d-o Have you taken a look at the learning rate? This was a problem for me since the default setting is a learning rate with exponential decay so my learning rate was quickly dropping to ~0

**Comment by johannes-do** [comment]
Has been quite a while ago, I don't think I have, could have been the problem. In the end I decided to create my own ssd from scratch ;).

**Comment by gul98** [comment]
can i have a look at your SSD. 
I am getting 0% accuracy in this code.

**Comment by dlazares** [comment]
@gul98  At the suggestion of someone else, I moved to the [google tensorflow object detection API](https://github.com/tensorflow/models/tree/master/object_detection). The process was easy and decently well documented.

**Comment by congjianting** [comment]
@gul98  i also face the same thing, the loss always keeps 40, did you find some solution?

**Comment by gul98** [comment]
no, my accuracy never got more than 33%

**Comment by satyajithj** [comment]
This is closed.
Was the problem solved?
